### PR TITLE
Add ReviewDrafter handler

### PR DESCRIPTION
**Pull request by ludeeus**
This implements the bot logic needed for the upcoming change to PR drafting.

### CODE REVIEW

File: services/bots/src/github-webhook/handlers/review_drafter.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+} from '@octokit/webhooks-types';
+import { EventType, HomeAssistantRepository } from '../github-webhook.const';
+import { WebhookContext } from '../github-webhook.model';
+import { BaseWebhookHandler } from './base';
+
+const MESSAGE_ID = '<!-- ReviewDrafterComment -->';
+
+const REVIEW_COMMENT = `${MESSAGE_ID}
+Please take a look at the requested changes, and use the **Ready for review** button when you are done, thanks :+1:
+For more information about this flow [take a look at this blog post](replace_with_blog_post_when_it_exist).
```
Comment (ludeeus) [inline services/bots/src/github-webhook/handlers/review_drafter.ts:13]: Note to self: The URL needs to be changed.

File: services/bots/src/github-webhook/handlers/review_drafter.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
```
Comment (balloob) [inline services/bots/src/github-webhook/handlers/review_drafter.ts:13]: Should this be on a new line? 
```suggestion

For more information about this flow [take a look at this blog post](replace_with_blog_post_when_it_exist).
```

File: services/bots/src/github-webhook/handlers/review_drafter.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  PullRequestReviewSubmittedEvent,
+} from '@octokit/webhooks-types';
+import { EventType, HomeAssistantRepository } from '../github-webhook.const';
+import { WebhookContext } from '../github-webhook.model';
+import { BaseWebhookHandler } from './base';
+
+const MESSAGE_ID = '<!-- ReviewDrafterComment -->';
+
+const REVIEW_COMMENT = `${MESSAGE_ID}
+Please take a look at the requested changes, and use the **Ready for review** button when you are done, thanks :+1:
```
Comment (balloob) [inline services/bots/src/github-webhook/handlers/review_drafter.ts:12]: I would explain a bit more what happened.

> Your pull request has been marked as "request for changes" and has automatically been marked as draft. Please address the comments and use the **Ready for review** button when you are done, thanks :+1:

File: services/bots/src/github-webhook/handlers/review_drafter.ts (hunk 2 above)
Comment (frenck) [reply services/bots/src/github-webhook/handlers/review_drafter.ts:13]: With URL

Should this be on a new line?  Also let's make it cursive.

```suggestion

[_Learn more about our pull request process._](https://developers.home-assistant.io/blog/2023/02/07/introducing-PR-drafting-in-reviews)
```

File: services/bots/src/github-webhook/handlers/review_drafter.ts (hunk 1 above)
Comment (frenck) [reply services/bots/src/github-webhook/handlers/review_drafter.ts:13]: Done! <https://developers.home-assistant.io/blog/2023/02/07/introducing-PR-drafting-in-reviews>

File: services/bots/src/github-webhook/handlers/review_drafter.ts (hunk 3 above)
Comment (ludeeus) [reply services/bots/src/github-webhook/handlers/review_drafter.ts:12]: That's the goal of the linked blogpost?
This (repeating) comment should IMO be short and link to more information.

Comment (frenck) [reply services/bots/src/github-webhook/handlers/review_drafter.ts:12]: Yeah agree, in longer term we could replace the URL with a more extensive/dedicated page even (that covers more subjects). But keeping the comment short and to the point is probably not a bad idea. (For the same reason we don't let netlify or codecov post comments).

Comment (balloob) [reply services/bots/src/github-webhook/handlers/review_drafter.ts:12]: ok 👍

**Review by frenck — APPROVED** [review]
LGTM ❤️ 

Thanks @ludeeus!

**Pull request merged**